src/clip.sol

```diff
@@ -337,17 +349,19 @@ contract Clipper {
     // Externally returns boolean for if an auction needs a redo
     function needsRedo(uint256 id) external view returns (bool) {
```

Comment:
The main reason I refactored `done` was because we needed the `price` as an input which is also calculated using the `tic` and `top` (the other 2 params). So IMO made more sense to calculate everything within the function and also return the price.
Let's see what the others think about it, but yeah I agree we could change it to `getStatus` and return both.
I'm also thinking we might want to change the `done` boolean to the opposite thing, so returns `true` when running and `false` when needs to be reset.